app/display.py

```diff
@@ -0,0 +1,23 @@
+from app.book import Book
+
+
+class BookDisplay:
+    @staticmethod
```

Comment:
The Single Responsibility Principle (SRP) is violated here. The BookDisplay class is responsible for both displaying and printing the books. It would be better to separate these responsibilities into two different classes.